sympy/assumptions/ask.py

```diff
@@ -142,7 +142,7 @@ def ask(proposition, assumptions=True, context=global_assumptions):
     if res is not None:
         return bool(res)
 
-    if assumptions == True:
+    if assumptions is True:
```

Comment:
Why not just ```if assumptions:``` ?

Also, ```assumptions``` is the result of a ```to_cnf``` call, so it might be an unevaluated ```And``` object, which certainly is not going to be ```is True```.
I see all tests pass, but maybe that's just because the code after this test comes to the same conclusion, while just taking some extra hoops. (Or maybe the ```is``` test would be faster, if the code inside this function is faster than the ``bool()``` member of ```And```.)
tl;dr: Just looking at the tests is not enough, unless the tests specifically cover the case that the CNF is an ```And```ed compound.